force-app/main/default/classes/PS_Request.cls

```diff
@@ -258,7 +260,13 @@ public inherited sharing class PS_Request {
                 if (String.isNotBlank(commitmentId)) {
                     value += '/' + commitmentId;
                 }
-
+            } else if (endpoint == ElevateEndpoint.COMMITMENT_PAUSE) {
+                value = String.format(
+                    ENDPOINT_COMMITMENT_CANCEL,
```

Comment:
Should this be `ENDPOINT_COMMITMENT_PAUSE`?

Reply:
Updated!